vpc-reachability-analyzer/elastio-vpc-reachability-analyzer.sh

```diff
@@ -1,26 +1,62 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+
+paragraph=$(cat <<'EOP'
+This script automates the process of evaluating the subnets within a VPC to
+test if they have the necessary access to be used with Elastio.
+
+To use it, choose the VPC you want to evaluate from the list, and the script
+will test each subnet in that VPC, one by one, by performing a Network
+Reachability Analyzer analysis.
+
+NOTE: This script will launch one very short-lived t2.micro instance in each
+subnet of the VPC, and it will perform Network Reachability Analyzer analysis,
+both of which incurr AWS charges. These charges are very small, but they will
+be incurred.
+
+Please also note that this script only works within the AWS environment and
+requires appropriate permissions to create and manage EC2 instances and network
+paths. Ensure you have these permissions before running the script to avoid
+execution errors.
+
+EOP
+)
+
+# Use 'fmt' to wrap the text to the current terminal width
+# Fall back to regular echo if that's not available
+echo "$paragraph" | fmt -w "$(tput cols)" || echo "${paragraph}"
+
+region=$(aws configure get region)
```

Comment:
Cannot run this script in CloudShell, it exits on this line. `aws configure get region` returns null in CloudShell.